stable/drupal/README.md

```diff
@@ -88,9 +90,17 @@ $ helm install --name my-release -f values.yaml stable/drupal
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
-## Persistence
+## Image
```

Comment:
This section seems redundant to me, as it's in the configuration table and there isn't really anything further to explain about this value?